src/app/pages/PerpetualPage/components/OrderHistoryTable/OrderHistoryRow.tsx

```diff
@@ -99,14 +113,12 @@ export const OrderHistoryRow: React.FC<OrderHistoryRowProps> = ({
         )}
       </td>
       <td>
-        <AssetValue
-          value={execSize}
-          assetString={pair?.baseAsset}
-          mode={AssetValueMode.auto}
-        />
-      </td>
-      <td>
-        {execPrice && (
+        {((tradeType === PerpetualTradeType.LIMIT ||
+          tradeType === PerpetualTradeType.STOP) &&
+          orderState === OrderState.Opened) ||
+        !execPrice ? (
+          <span className="tw-text-2xl">-</span>
```

Comment:
I would not use a different font size, but rather an en-dash
```suggestion
          <span>–</span>
```